addons/web_editor/static/src/js/wysiwyg/widgets/image_optimize_dialog.js

```diff
@@ -303,6 +306,18 @@ var ImageOptimizeDialog = Dialog.extend({
         ev.preventDefault();
         this.$widthInput.val(parseInt($(ev.target).data('width')));
         this._onWidthInput();
+        this.$customSizeBtn.removeClass('active');
+        this.$customSizeInputs.addClass('d-none');
+    },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onCustomSizeBtnClick: function (ev) {
+        ev.preventDefault();
+        this.$widthPresets.removeClass('active');
+        this.$customSizeBtn.addClass('active');
+        this.$customSizeInputs.removeClass('d-none');
```

Comment:
All of the new system you introduced could probably avoid all this JS code and use standard bootstrap behavior (toggling pane contents etc). Did you try ?

Reply:
I'm using now collapse system instead (https://getbootstrap.com/docs/4.3/components/collapse/)